homeassistant/components/hunterdouglas_powerview/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Hunter Douglas PowerView",
```

Comment:
You can omit this. It's the same as the manifest.

Reply:
done